docs_user/modules/proc_configuring-openshift-worker-nodes.adoc

```diff
@@ -1,18 +1,16 @@
 [id="configuring-openshift-worker-nodes_{context}"]
 
-= Configuring {OpenShiftShort} worker nodes
+= Configuring isolated networks on {OpenShiftShort} worker nodes
 
-{rhocp_long} worker nodes that run {rhos_prev_long} services need a way to connect the service
-pods to isolated networks. This requires physical network configuration on the
-hypervisor.
+To connect service pods to isolated networks on {rhocp_long} worker nodes that run {rhos_prev_long} services physical network configuration on the hypervisor is required.
 
-This configuration is managed by the NMState operator, which uses the custom resources (CRs) to
-define the desired network configuration for the nodes.
+This configuration is managed by the NMState operator, which uses custom resources (CRs) to define the desired network configuration for the nodes.
```

Comment:
Does the NMState operator use the `NodeNetworkConfigurationPolicy` CR, specifically?

Reply:
Yep!